tests/Makefile

```diff
@@ -37,11 +43,24 @@ gcs:
 	wait $$FAKE_GCS_SERVER_PID 2>/dev/null || true; \
 	rm -rf ${FAKE_GCS_SERVER_ROOT}
 
+azure:
+	@echo "Starting Azurite server on port ${AZURITE_API_PORT}"; \
+	azurite --silent --skipApiVersionCheck --location /tmp/azurite --blobPort ${AZURITE_API_PORT} & \
+	AZURITE_PID=$$!; \
+	echo "Azurite PID is $$AZURITE_PID"; \
+	RUNAI_STREAMER_LOG_LEVEL=INFO RUNAI_STREAMER_LOG_FILE=/tmp/azure_cpp.log AZURE_STORAGE_CONNECTION_STRING="DefaultEndpointsProtocol=http;AccountName=${AZURITE_ACCOUNT_NAME};AccountKey=${AZURITE_ACCOUNT_KEY};BlobEndpoint=http://127.0.0.1:${AZURITE_API_PORT}/${AZURITE_ACCOUNT_NAME};" AZURE_CONTAINER=${AZURITE_BUCKET} pytest -v azure/ 2>&1 | tee /tmp/azure_test.log; \
```

Comment:
I wonder if we really need to enable the cpp logs on the tests.
But I guess no harm would happen if it is.